env/packages.yml

```diff
@@ -71,6 +71,9 @@
 
     - name: Compile gRPC and its dependencies
       shell: make -j{{ ansible_processor_vcpus }} chdir=/tmp/grpc
+      environment:
+        - CC: gcc-6
+        - CXX: g++-6
```

Comment:
gRPC in the current docker image is built by g++ 6. Since the CI builds are fine with that, it should be okay to use g++ 6.